### PR TITLE
DOC: Added two pyart examples (VAD and hydrometeors)

### LINKED ISSUE #1260: ENH: Add a new example showing hydrometeor classification and QPE functionality
At the recent ERAD tutorials, there were some great new tutorials put together by the meteoswiss/meteofrance team (@wolfidan @jfigui )

[Sample notebook](https://openradarscience.org/erad2022/notebooks/pyart/exercice1_swiss_thunderstorm.html)

It would be nice to have a concise example of this in our examples gallery!

### PR DESCRIPTION
**Pull request by wolfidan**
Closes #1260 

Dear pyart team,

Sorry for the long delay, I had to wait for the green light to share some of the data and had some fieldwork to do...

Here are two new examples for the pyart gallery taken from the erad2022 open radar course.

**plot_vad.py:**
Compute VAD for every sweep of a volumetric scan, compute a vertical profile of U and V wind components, display the results

**plot_hydrometeors.py:**
Compute attenuation and KDP, read temperature from another source and add it to radar object, compute hydrometeors, display the results on a PPI plot with a categorical colorbar

It requires three files to be acquired with the *get_test_data* method. I hosted the files [here ](https://www.swisstransfer.com/d/7d632b77-c120-4316-be80-13a3a750aca6). Could you please put them on https://adc.arm.gov/pyart/example_data ? Tell me if you'd prefer me to send them to you in some other way.

Thank you very much,
Cheers,
Daniel

### CODE REVIEW

**Comment by mgrover1** [comment]
Thanks @wolfidan !! How large are these files?

**Comment by wolfidan** [comment]
@mgrover1, 237 Kb, 27 Mb and 1.4 Mb

**Comment by zssherman** [comment]
Added PEP8 changes and open radar data install to CI

File: examples/retrieve/plot_hydrometeor.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+# Author: Daniel Wolfensberger (daniel.wolfensberger@meteoswiss.ch)
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+
+import pyart
+from pyart.testing import get_test_data
```
Comment (mgrover1) [inline examples/retrieve/plot_hydrometeor.py:18]:
```suggestion
from open_radar_data import DATASETS
```

File: examples/retrieve/plot_hydrometeor.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+# Read in a sample file
+filename = get_test_data('MLL2217907250U.003.nc')
```
Comment (mgrover1) [inline examples/retrieve/plot_hydrometeor.py:21]:
```suggestion
filename = DATASETS.fetch('MLL2217907250U.003.nc')
```

File: examples/retrieve/plot_hydrometeor.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+radar = pyart.io.read_cfradial(filename)
+
+# Read temperature preinterpolated from NWP model
+filename = get_test_data('20220628072500_savevol_COSMO_LOOKUP_TEMP.nc')
```
Comment (mgrover1) [inline examples/retrieve/plot_hydrometeor.py:25]:
```suggestion
filename = DATASETS.fetch('20220628072500_savevol_COSMO_LOOKUP_TEMP.nc')
```

File: examples/retrieve/plot_vad.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+"""
+
+# Author: Daniel Wolfensberger (daniel.wolfensberger@meteoswiss.ch)
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import pyart
+from pyart.testing import get_test_data
```
Comment (mgrover1) [inline examples/retrieve/plot_vad.py:16]:
```suggestion
from open_radar_data import DATASETS
```

File: examples/retrieve/plot_vad.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+# Read in a sample file
+filename = get_test_data('MLA2119412050U.nc')
```
Comment (mgrover1) [inline examples/retrieve/plot_vad.py:19]:
```suggestion
filename = DATASETS.fetch('MLA2119412050U.nc')
```

**Review by mgrover1 — COMMENTED** [review]
We need to read from the open_radar_data package

**Comment by mgrover1** [comment]
@zssherman we also need the documentation environment to be updated

**Review by mgrover1 — APPROVED** [review]
Looks good!

**Pull request merged**